npm/pkg/controlplane/controllers/v2/podcontroller.go

```diff
@@ -392,15 +389,8 @@ func (c *PodController) syncAddedPod(podObj *corev1.Pod) error {
 		allSets := []*ipsets.IPSetMetadata{targetSetKey, targetSetKeyValue}
 
 		klog.Infof("Creating ipsets %v if it does not already exist", allSets)
-		c.dp.CreateIPSets(allSets)
-
-		klog.Infof("Adding pod %s to ipset %s", npmPodObj.PodIP, labelKey)
-		if err = c.dp.AddToSets([]*ipsets.IPSetMetadata{targetSetKey}, podMetadata); err != nil {
-			return fmt.Errorf("[syncAddedPod] Error: failed to add pod to label ipset with err: %w", err)
-		}
-
-		klog.Infof("Adding pod %s to ipset %s", npmPodObj.PodIP, podIPSetName)
-		if err = c.dp.AddToSets([]*ipsets.IPSetMetadata{targetSetKeyValue}, podMetadata); err != nil {
+		klog.Infof("Adding pod %s to ipset %s and %s", npmPodObj.PodIP, targetSetKey, podIPSetName)
```

Comment:
We print `*IPSetMetadata` with `%s`
```go
klog.Infof("Adding pod %s (ip : %s) to ipset %s and %s", npmPodObj.Name, npmPodObj.PodIP, labelKey, podIPSetName)
```

Would like to locate codes in liine 385 - 388.
```go
targetSetKey := ipsets.NewIPSetMetadata(labelKey, ipsets.KeyLabelOfPod)
podIPSetName := util.GetIpSetFromLabelKV(labelKey, labelVal)
targetSetKeyValue := ipsets.NewIPSetMetadata(podIPSetName, ipsets.KeyValueLabelOfPod)
```